js/interactive-guides/step-content.js

```diff
@@ -262,6 +262,35 @@ var stepContent = (function() {
     createEndOfGuideContent();
   };
 
+  var calculateWidgetHeight = function(percentageHeight) { 
+    var widgetHeight = "300px";
+    var rightColumn = $("#code_column:visible");
+    if (rightColumn) {
```

Comment:
rightColumn is a jQuery object.   So the check here should be 
if (rightColumn.length > 0) {